README.md

```diff
@@ -51,6 +51,15 @@ make all
 
 are both valid. To see a list of all valid targets, run `make help`.
 
+**Python version notes:**
+1. by default the python target runs tests on all supported python versions.
+   To run tests on just specific python version(s), specify the `PYTESTENV`
+   environment variable, e.g., `PYTESTENV=py27,py35 make python`.
+2. by default the code generator is run on the system's default python version.
+   To use a different version, specify the `TOXENV` environment variable, e.g.,
+   `TOXENV=py27 make all`
```

Comment:
If there is a particular supported set of python versions for the generator, make sure to mention it here.  Do we want to support it for an arbitrary python version?

Reply:
Currently running the generator isn't limited to certain python versions but only certain versions are guaranteed to work. I'll improve the docs a bit.